src/contracts/CLSynchronicityPriceAdapterBaseToPeg.sol

```diff
@@ -78,6 +89,8 @@ contract CLSynchronicityPriceAdapterBaseToPeg is ICLSynchronicityPriceAdapter {
       return 0;
     }
 
-    return (assetToPegPrice * int256(10 ** DECIMALS)) / (baseToPegPrice);
+    return
+      (assetToPegPrice * int256(10 ** DECIMALS) * int256(10 ** BASE_TO_PEG_DECIMALS)) /
```

Comment:
Can this create problems of precision whenever `BASE_TO_PEG_DECIMALS` is way bigger than `ASSET_TO_PEG_DECIMALS`?